pkg/vault/client.go

```diff
@@ -57,14 +57,7 @@ func initClient(vc *vaultClient) error {
 	}
 
 	token, err := vc.authClient.GetVaultToken(vc)
-	if err != nil {
```

Comment:
why was this removed / .SetToken being called even if there's an error getting the token?

Reply:
Good call out, I was just being lazy to get tests passing.  Re-added and tests are passing now!